src/game_stats.py

```diff
@@ -57,10 +59,34 @@ def log_end_game(self):
     def log_failed_run(self):
         self._runs_failed += 1
 
-    def get_current_game_length(self):
+    def pause_timer(self):
+        """ Pauses the timer """
+        if self._timer is None:
+            return
+        if self._paused:
+            return
+        Logger.info(f'Pausing timer')
```

Comment:
Make this a debug message maybe. There is already a info message in bot.py when it gets paused